notebooks/official/automl/sdk_automl_video_classification_batch.ipynb

```diff
@@ -594,7 +595,8 @@
       "source": [
         "### Get test item(s)\n",
         "\n",
-        "Now do a batch prediction to your Vertex AI model. You use arbitrary examples from the dataset as a test items. Don't be concerned that the examples were likely used in training the model as this tutorial is just about how to make a batch prediction."
+        "Now do a batch prediction to your Vertex AI model. Use arbitrary examples from the dataset as a test items.\n",
```

Comment:
Should 'a' be removed before 'test items'?

Reply:
yes

Reply:
done. and re-committed.